tasks/Qc.wdl

```diff
@@ -429,6 +429,7 @@ task RxIdentifier {
   }
   output {
     File rx_result = "~{output_name}"
+    String rx_value = read_tsv(output_name)[0][5]
```

Comment:
Is the Rx value the only value being reported? I'm sure users would also want to know the classification, ie. XX, XY, or ambiguous. To be thorough I'd also suggest either the confidence interval or the range +/- the confidence interval. The floor/ceiling of the +/- confidence interval is what's ultimately used to determine the classification (if I recall correctly).

Reply:
Ah I see, you are reporting the classification only... that may be sufficient as our goal is to keep the output values limited. +1 as-is.

Reply:
I guess I should've named that "classification" instead of "value".  We could pretty easily add any of the other columns as additional outputs if they'd be useful at some point--even if they are outputs from the step we don't have to report them in the data table necessarily.